### DISCUSSION TITLE
2005/05/07/%E6%80%8E%E6%A0%B7%E5%81%9A%E7%95%99%E8%A8%80%E6%9D%BF

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#22] [Announcements]
# 2005/05/07/%E6%80%8E%E6%A0%B7%E5%81%9A%E7%95%99%E8%A8%80%E6%9D%BF

Step1，新建一项网络日志用于做留言板。

https://blog.qizhen.xyz/2005/05/07/%E6%80%8E%E6%A0%B7%E5%81%9A%E7%95%99%E8%A8%80%E6%9D%BF

**Comment by ruanqizhen** [comment]
zero
2005年09月25日 22:05 编辑
按照这个方法~结果是——成功!呵呵~~谢谢

回复
菲飞
2005年07月26日 03:56 编辑
哇塞!!!真的是很棒哎 ! 谢谢咯 我可是要慢慢研究咯

回复
zim
2005年07月25日 21:07 编辑
呵呵谢谢`大师

回复
西骑骨怪
2005年05月21日 13:51 编辑
第二步的那个固定链接是怎么弄的啊？我不是太懂咯。谢谢了。

回复
小月亮
2005年05月20日 10:58 编辑
要如何设计啊？做是做好了

回复
Qizhen
2005年05月8日 10:49 编辑
I can\’t see any problems for your reading Chinese. However, I\’ll translate it into English in a couple of days. ^_^

回复
blue
2005年05月8日 02:22 编辑
呃……可以把它翻译成英语吗？？拜托……T~T

回复
j
2005年05月7日 10:43 编辑
hi~我是一个BEGINNER，如何才能让我的界面看起来更漂亮一些她现在有点难看谢谢

回复

**Comment by ruanqizhen** [comment]
ruby.chen
2005年10月15日 13:20 编辑
我哭泣了!!!!

回复
ruby.chen
2005年10月15日 13:20 编辑
为什么我进不去我的空间啊!!!!

回复
多多
2005年10月6日 16:54 编辑
呵呵，我来也

回复
alex
2005年09月9日 23:36 编辑
谢谢咯

回复
fan
2005年08月25日 14:29 编辑
留个脚印

回复
匕禾页
2005年08月3日 23:22 编辑
你真是个热心的人，谢谢啦！

回复
健
2005年07月31日 23:09 编辑
太好了,佩服~~

回复
Unknown
2005年07月25日 21:03 编辑
字有点小,看不清耶,能不能拜托弄大点呢?

回复
rOckbeAr
2005年06月5日 11:45 编辑
十分十分的感谢

回复
Radon
2005年06月3日 12:13 编辑
厉害哦，真的很专业耶……我那个就不行了，空白的要命！有空再学习一下……呵呵

回复
Unknown
2005年05月27日 18:39 编辑
你好专业哦，我正在建设中，一定要用IE吗？

回复
天水崖
2005年05月26日 08:54 编辑
学一下

回复
Jessica
2005年05月15日 11:31 编辑
牛X 学习一下。

回复
jean
2005年05月10日 10:13 编辑
you are the first one who come to my MSN space,welcome!And ,i will see you .

回复
妖。
2005年05月8日 22:09 编辑
哈哈….我也来你这啦~~~~很高兴认识你

回复
